app/views/layouts/application.html.erb

```diff
@@ -25,25 +25,9 @@
   <body class="mainstream <%= yield :body_classes %>">
     <div id="wrapper" class="<%= wrapper_class(@publication || @presenter) %>">
       <% unless current_page?(root_path) %>
-        <% if present_new_navigation? %>
```

Comment:
Spoke to Imeh, Mia and Alex about what we should do regarding the banner. Have concluded that we will remove this beta banner and survey link (as you have done here, and will do in a new PR in government-frontend), and use a new survey on our new tests.